diff --git a/android.sh b/android.sh
index 787fd25..2eb9cd7 100755
--- a/android.sh
+++ b/android.sh
@@ -133,13 +133,17 @@ if [[ -z ${BUILD_VERSION} ]]; then
   exit 1
 fi
 
+export SKIP_openh264=1
+
 # PROCESS FULL OPTION AS LAST OPTION
 if [[ -n ${BUILD_FULL} ]]; then
   for library in {0..58}; do
-    if [ ${GPL_ENABLED} == "yes" ]; then
+
+    # 32 is OpenH264
+    if [[ ${GPL_ENABLED} == "yes" && $library -ne 32 ]]; then
       enable_library "$(get_library_name $library)" 1
     else
-      if [[ $(is_gpl_licensed $library) -eq 1 ]]; then
+      if [[ $(is_gpl_licensed $library) -eq 1 && $library -ne 32 ]]; then
         enable_library "$(get_library_name $library)" 1
       fi
     fi
