diff --git a/android.sh b/android.sh
index bf94df5..401daa3 100755
--- a/android.sh
+++ b/android.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export SKIP_ffmpeg_kit=1
+
 if [[ -z ${ANDROID_SDK_ROOT} ]]; then
   echo -e "\n(*) ANDROID_SDK_ROOT not defined\n"
   exit 1
diff --git a/scripts/android/ffmpeg.sh b/scripts/android/ffmpeg.sh
index 1ee0277..c0074df 100755
--- a/scripts/android/ffmpeg.sh
+++ b/scripts/android/ffmpeg.sh
@@ -394,14 +394,25 @@ fi
   --nm="${NM}" \
   --extra-libs="$(pkg-config --libs --static cpu-features)" \
   --target-os=android \
+  --disable-programs \
+  --disable-doc \
+  --disable-swscale \
+  --disable-postproc \
+  --disable-avdevice \
+  --disable-network \
+  --disable-avfilter \
+  --disable-everything \
+  --enable-decoder=hevc \
+  --enable-decoder=pcm_s16le \
+  --enable-encoder=aac \
+  --enable-encoder=hevc \
+  --enable-muxer=mp4 \
   ${ASM_OPTIONS} \
   --enable-cross-compile \
   --enable-pic \
   --enable-jni \
   --enable-optimizations \
-  --enable-swscale \
   ${BUILD_LIBRARY_OPTIONS} \
-  --enable-v4l2-m2m \
   --disable-outdev=fbdev \
   --disable-indev=fbdev \
   ${SIZE_OPTIONS} \
