diff --git a/scripts/android/ffmpeg.sh b/scripts/android/ffmpeg.sh
index 8b311fd..5d01c4c 100755
--- a/scripts/android/ffmpeg.sh
+++ b/scripts/android/ffmpeg.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export SKIP_ffmpeg_kit=0
+
 HOST_PKG_CONFIG_PATH=$(command -v pkg-config)
 if [ -z "${HOST_PKG_CONFIG_PATH}" ]; then
   echo -e "\n(*) pkg-config command not found\n"
@@ -423,6 +425,19 @@ fi
   --arch="${TARGET_ARCH}" \
   --cpu="${TARGET_CPU}" \
   --target-os=android \
+  --disable-programs \
+  --disable-doc \
+  --disable-postproc \
+  --disable-network \
+  --disable-everything \
+  --enable-decoder=hevc \
+  --enable-decoder=pcm_s16le \
+  --enable-encoder=aac \
+  --enable-encoder=hevc \
+  --enable-muxer=mp4 \
   ${ASM_OPTIONS} \
   --ar="${AR}" \
   --cc="${CC}" \
@@ -436,10 +451,8 @@ fi
   --enable-pic \
   --enable-jni \
   --enable-optimizations \
   ${BUILD_LIBRARY_OPTIONS} \
   --enable-pthreads \
-  --enable-v4l2-m2m \
   --disable-outdev=fbdev \
   --disable-indev=fbdev \
   ${SIZE_OPTIONS} \
